Build/Build.cs

```diff
@@ -302,9 +302,48 @@ from framework in supportedFrameworks
             ReportTestOutcome(projects.Select(p => $"*{p.Name}*.trx").ToArray());
         });
 
+    Target TestingPlatformFrameworks => _ => _
```

Comment:
🔧You must have not tried to run this locally, since it fails

Reply:
Ah sorry. Let me take a look tomorrow.

Reply:
Sorry for the delay. Passes for me locally now.